runtime/runtime.go

```diff
@@ -425,14 +428,19 @@ func (rt *Runtime) Serve(ctx context.Context) error {
 		return fmt.Errorf("at least one address must be configured in runtime parameters")
 	}
 
+	publicInterfaceWarning := ""
+	if !rt.Params.AddrSetByUser {
+		publicInterfaceWarning = " OPA is running on a public (0.0.0.0) network interface. Unless you intend to expose OPA outside of the host, binding to the localhost interface (--addr localhost:8181) is recommended. See https://www.openpolicyagent.org/docs/latest/security/#interface-binding"
+	}
+
```

Comment:
What about something like:

```go
message := "Initializing server."

if !rt.Params.AddrSetByUser {
    message += " OPA is running on a public......"
}
```

Then 

```go
rt.logger.WithFields(map[string]interface{}{
		"addrs":            *rt.Params.Addrs,
		"diagnostic-addrs": *rt.Params.DiagnosticAddrs,
	}).Info(message)
```

Reply:
Ah yeah @ashutosh-narkar this makes much more sense, thanks! 👍